src/app/global.js

```diff
@@ -57,6 +57,6 @@ var
 
     // Torrent engines
     peerflix = require('peerflix'),
-
+    WebTorrent = require('webtorrent'),
```

Comment:
We will need it with the new streamer running on Webtorrent Please see #424  #470